amoro-ams/src/main/java/org/apache/amoro/server/table/TableManager.java

```diff
@@ -33,6 +35,9 @@ public interface TableManager extends TableRuntimeHandler {
 
   TableRuntime getRuntime(Long tableId);
 
+  /** Get the count of table runtimes associated to the given filter. */
+  int getRuntimeCount(@Nullable String dbFilter, @Nullable String tableFilter);
```

Comment:
I think instead of defining such an interface, it would be better to directly define a listRuntimes interface.